waiter/src/waiter/core.clj

```diff
@@ -804,7 +813,8 @@
                      scaling/scale-app service-id->service-description-fn router-state-push-mult)))
    :autoscaling-multiplexer (pc/fnk [[:routines delegate-instance-kill-request-fn peers-acknowledged-blacklist-requests-fn]
                                      [:settings [:scaling inter-kill-request-wait-time-ms] blacklist-config]
-                                     [:state instance-rpc-chan scheduler]]
+                                     [:state instance-rpc-chan]
+                                     [:scheduler scheduler]]
```

Comment:
Nitpick: lexicographic order on the keys (`:scheduler` before `:settings`).

Reply:
fixed